app/views/spree/checkout/payment/_gateway.html.erb

```diff
@@ -7,7 +7,7 @@
     id: "name_on_card_#{payment_method.id}",
     label: t('spree.name_on_card'),
     name: "#{param_prefix}[name]",
-    value: @order.billing_name
+    value: "#{@order.billing_firstname} #{@order.billing_lastname}"
```

Comment:
We were thinking that this extension would support Solidus starting from v2.11, which is the current master. It would be awesome to make it work with v2.10 as well though, at least until we release v2.11. What about supporting both versions conditionally using one or the other version checking if `@order` responds to `billing_name`?

Reply:
Sounds good to me, and yeah, we'd love it to support v2.10 -- let us know how we can help.